src/embed-receipt.ts

```diff
@@ -183,15 +183,15 @@ export class RWPEmbedReceipt extends LitElement {
       datapackageHash,
       publicKey,
       software,
-    } = this.collInfo?.verify || {};
+    } = this.collInfo?.verify ?? {};
 
     const sourceUrl = this.collInfo?.sourceUrl;
 
     const certFingerprintUrl = certFingerprint
       ? `https://crt.sh/?q=${certFingerprint}`
       : "";
 
-    const dateStr = tsToDate(this.ts).toLocaleString();
+    const dateStr = tsToDate(this.ts!).toLocaleString();
```

Comment:
Will this throw if `this.ts` is missing for whatever reason? Maybe we can handle this in the template, `if (!this.ts) return nothing`

Reply:
Good question! It actually looks like `tsToDate` already returns an empty string if passed a falsy value, lemme just update that type.